src/nnet3/nnet-training.cc

```diff
@@ -222,11 +224,10 @@ void NnetTrainer::PrintMaxChangeStats() const {
 void ObjectiveFunctionInfo::UpdateStats(
     const std::string &output_name,
     int32 minibatches_per_phase,
-    int32 minibatch_counter,
```

Comment:
I think the previous method was wrong, since it assumes single output in egs. The phase should be computed w.r.t num_minibatches per output not total num_minibatches.